mods/voidmulti.sc2mod/base.sc2data/gamedata/upgradedata.xml

```diff
@@ -692,7 +690,7 @@
         <Race value="Zerg"/>
         <ScoreAmount value="200"/>
         <ScoreResult value="BuildOrder"/>
-        <EffectArray Operation="Set" Reference="Unit,Hydralisk,Speed" Value="2.812500"/>
+        <EffectArray Reference="Unit,Hydralisk,Speed" Value="0.750000"/>
```

Comment:
Muscular Augments move speed bonus increased from 0.79 to 1.05 (from 0.5625 to 0.75 on normal speed)

Hydralisk base move speed is 2.25. The upgrade used to set it to 2.8125 which means it added 0.5625.
Now it adds 0.75 for an upgraded speed of 3.0.
This converted to normal game speed results in the values the patch notes state